hw_diag/templates/diagnostics_page_light_miner.html

```diff
@@ -0,0 +1,156 @@
+<!doctype html>
```

Comment:
added a new copy of template because layout has to be modified to make it symmetric. Trying to do layout with lot of if else was becoming unreadable.